### DISCUSSION TITLE
Welcome to actix-web Discussions!

### DISCUSSION THREAD

**Discussion by JohnTitor** [actix/actix-web#1769] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robjtede** [comment]
I'm coming around to this since you can convert issues to discussions. Added it to the issue template chooser. https://github.com/actix/actix-web/issues/new/choose
